src/zig_llvm.h

```diff
@@ -406,6 +406,39 @@ enum ZigLLVM_ObjectFormatType {
     ZigLLVM_Wasm,
 };
 
+#define ZigLLVM_DIFlags_Zero 0U
```

Comment:
Can you add static_casts in zig_llvm.cpp to detect if these values change? Or do we have some reason to believe these values are ABI-stable?